PathfinderAPI/Replacements/ContentLoader.cs

```diff
@@ -711,32 +711,32 @@ static ContentLoader()
             executor.RegisterExecutor("Computer.DLCCredits", (exec, info) =>
             {
                 DLCCreditsDaemon credits;
-                string title, text = null;
-                if (info.Attributes.TryGetValue("Title", out title) || info.Attributes.TryGetValue("ButtonText", out text))
+                string title = info.Attributes.TryGetValue("Title", out string titleValue) ? titleValue : null;
+                string text = info.Attributes.TryGetValue("ButtonText", out string textValue) ? textValue : null;
+                if (title != null || text != null)
                 {
                     credits = new DLCCreditsDaemon(comp, os, title.Filter(), text.Filter());
                 }
                 else
                 {
                     credits = new DLCCreditsDaemon(comp, os);
                 }
-                
                 credits.ConditionalActionsToLoadOnButtonPress = info.Attributes.GetString("ConditionalActionSetToRunOnButtonPressPath", null);
                 comp.daemons.Add(credits);
             });
             executor.RegisterExecutor("Computer.CreditsDaemon", (exec, info) =>
             {
                 DLCCreditsDaemon credits;
-                string title, text = null;
-                if (info.Attributes.TryGetValue("Title", out title) || info.Attributes.TryGetValue("ButtonText", out text))
+                string title = info.Attributes.TryGetValue("Title", out string titleValue) ? titleValue : null;
+                string text = info.Attributes.TryGetValue("ButtonText", out string textValue) ? textValue : null;
```

Comment:
Why didn't you just do `info.Attributes.TryGetValue("Title", out title) && info.Attributes.TryGetValue("ButtonText", out text)`?

Reply:
Because you would need to input Title and ButtonText obligatorily, and I wanted to keep the main concept. I will change it anyways

Reply:
The problem is you didn't change the subject at all, this would do absolutely nothing.

Reply:
It solved the Bug of the issue I attached

Reply:
Actually you can remove shortcircuting in C# by replacing `||` with `|`

Reply:
Ready, I changed it to AND (&&)